.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "all"
```

Comment:
All of these things can be done with eslint. I suggest to remove prettier and only stick to eslint.